fhir-operation/src/main/java/com/ibm/fhir/operation/registry/FHIROperationRegistry.java

```diff
@@ -46,7 +47,20 @@ private FHIROperationRegistry() {
                     log.severe("Operation $" + operationName + " has failed validation and will be skipped.");
                     continue;
                 }
-                operationMap.put(operation.getName(), operation);
+                List<ResourceType> operationResourceTypes = operation.getDefinition().getResource();
```

Comment:
I think a comment explaining the purpose of the generated map would help here. 
There is also some duplication of the code below, breaking out into a different method would be good (only pass in the unique key element)

Reply:
added comments for the map key.